test/UtilityRepository-test.js

```diff
@@ -0,0 +1,72 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+import UtilityRepository from '../src/UtilityRepository';
+
+let dataSleep, dataActivity, dataUser, utilityRepo;
+
+beforeEach(() => {
+  dataSleep = 
+    [
+      { userID: 44, date: "2019/08/16", hoursSlept: 9.9, sleepQuality: 2.8 },
+      { userID: 44, date: "2019/08/17", hoursSlept: 6.9, sleepQuality: 4.5 },
+      { userID: 45, date: "2019/08/18", hoursSlept: 7.3, sleepQuality: 2.1 },
+      { userID: 46, date: "2019/08/18", hoursSlept: 9.2, sleepQuality: 1.5 },
+      { userID: 45, date: "2019/08/19", hoursSlept: 7, sleepQuality: 4.2 },
+      { userID: 45, date: "2019/08/20", hoursSlept: 10.4, sleepQuality: 4.3 },
+      { userID: 45, date: "2019/08/21", hoursSlept: 7.8, sleepQuality: 1.7 },
+      { userID: 45, date: "2019/08/22", hoursSlept: 8.5, sleepQuality: 2.3 },
+      { userID: 45, date: "2019/08/23", hoursSlept: 10.9, sleepQuality: 2.3 },
+      { userID: 45, date: "2019/08/24", hoursSlept: 8.3, sleepQuality: 1.5 }
+    ];
+
```

Comment:
Very nice test file. One common convention you'll see in the real world is that people will put a directory inside of their `tests` directory called something like `mocks` and that will hold any mock datasets that they've created for testing. You would pull this `dataSleep` array out into its own file in that `mock` directory, and then import it into your test so that you have access to it as a variable rather than having the entire array cluttering up your test file.